scripts/format_notebooks.py

```diff
@@ -220,6 +225,7 @@ def preprocess_cell(self, cell, resources, cell_index):
     on_ci = args.ci
     format_code = args.format_code or args.default
     clear_warnings = args.clear_warnings or args.default
+    coalesce_streams = args.coalesce_streams or args.default
```

Comment:
I think this requires a corresponding update to the help text for the "default" option?